core/services/fluxmonitor/fetchers.go

```diff
@@ -175,7 +175,7 @@ func (m *medianFetcher) Fetch(meta map[string]interface{}) (decimal.Decimal, err
 		go func() {
 			price, err := fetcher.Fetch(meta)
 			if err != nil {
-				logger.Error(err)
+				logger.Warn(err)
```

Comment:
I think a better way to use the logger here is `Warnw("a desciption of the error, perhaps Fetch failed", "error", err)`

Reply:
the fetcher returns a descriptive error https://github.com/smartcontractkit/chainlink/blob/1262ba6f0592a7a2c2793897ea85c17d11bf09f0/core/services/fluxmonitor/fetchers.go#L66 - I could pull that up to this layer though?

Reply:
What would it look like?

Reply:
logger.Warnf("unable to fetch price from %s with payload '%s, err: %s'", p.url.String(), p.requestData, err)

Reply:
then in the http fetcher (non-median), it be just return decimal.Decimal{}, err

Reply:
I'd probably still use `Warnw` over `Warnf` better to use structure IMHO.

Reply:
actually that would make it inconsistent with the rest of the errors in ```func (p *httpFetcher) Fetch(meta map[string]interface{}) (decimal.Decimal, error) {
```

Reply:
I would say that `httpFetcher` is then inconsistent with the rest of the code base, but in that case I'll do a wider audit.